Added how to disable the lettuce health indicator if needed

Omasiri opened the pull request:
This adds information about how to disable the micronaut lettuce health indicator if needed. It is not very obvious how to do it, except by looking at the code.

Omasiri committed:
Update config.adoc

This adds information about how to disable the micronaut lettuce health indicator if needed. It is not very obvious how to do it, except by looking at the code.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/micronaut-projects/micronaut-redis?pullRequest=583) <br/>All committers have signed the CLA.

Omasiri committed:
Fix formatting in documentation

Update Redis health indicator documentation to include proper code block formatting

Omasiri committed:
Fix formatting for redis health config block

Update Redis health indicator documentation to include proper code block formatting

graemerocher commented on src/main/docs/guide/config.adoc:
````diff
@@ -80,6 +80,13 @@ include::{testsredis}/NamedCodecFactory.groovy[tags=namedCodec2, indent=0]
 When the `redis-lettuce` module is activated a api:configuration.lettuce.health.RedisHealthIndicator[] is activated resulting in the `/health` endpoint and https://docs.micronaut.io/latest/api/io/micronaut/health/CurrentHealthStatus.html[CurrentHealthStatus] interface resolving the health of the Redis connection or connections.
 
+The health indicator is enabled by default. To disable the health endpoint, you can do so via the config. 
+```
+redis:
+   health:
+    enabled: false
+```
````
```suggestion
[configuration]
----
redis:
   health:
    enabled: false
----
```

graemerocher committed:
Update src/main/docs/guide/config.adoc
